tests/services/inventory/test_data/mock_cai_access_policies.dump

```diff
@@ -0,0 +1,7 @@
+{"name":"//cloudresourcemanager.googleapis.com/organizations/111222333","asset_type":"cloudresourcemanager.googleapis.com/Organization","access_policy":{"name":"accessPolicies/111222333444","parent":"organizations/abc","title":"default policy"},"ancestors":["organizations/111222333"]}
+{"name":"//cloudresourcemanager.googleapis.com/organizations/111222333","asset_type":"cloudresourcemanager.googleapis.com/Organization","access_policy":{"name":"accessPolicies/111222","parent":"organizations/abc","title":"default policy"},"ancestors":["organizations/111222333"]}
```

Comment:
you can only have one access policy per org

Reply:
Yea it doesn't allow creation of second access policy in the same org. Removed second access policy and updated tests accordingly.